### PR TITLE
Tq/feature branch example

### PR DESCRIPTION
**Pull request by tatsqui**

### CODE REVIEW

File: garbage-humans.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+You all are great non-garbage humans
```
Comment (kanderson38) [inline garbage-humans.txt:1]: That is so nice!

File: barnacle.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+This is practice for PR'ing.
```
Comment (kanderson38) [inline barnacle.txt:1]: It sure is.

File: README.md [hunk 3]
```diff
@@ -16,7 +16,7 @@ Person C should modify content here.
 
 ## Person D:
 
-Person D should modify content here.
+I made two text files :-)
```
Comment (kanderson38) [inline README.md:19]: Hey another comment

**Review by kanderson38 — APPROVED** [review]

**Comment by kanderson38** [comment]
Looks great! SHIP IT

**Review by kanderson38 — COMMENTED** [review]
Here is just a general comment

File: barnacle.txt (hunk 2 above)
Comment (tatsqui) [reply barnacle.txt:1]: Yep.

File: garbage-humans.txt (hunk 1 above)
Comment (tatsqui) [reply garbage-humans.txt:1]: Dan said it! So it must true!!

**Pull request merged**